tests/files/test.jsonl

```diff
@@ -0,0 +1 @@
e: Grammy-nominated singer-songwriter Jewel is coming to NBC's cult-favorite a cappella competition series this winter. Jewel will join Boyz II Men's Shawn Stockman and Ben Folds as judges on the show, and Nick Lachey will return as host. The singer replaces exiting singer-songwriter Sara Bareilles on the show. \"There are very few musical artists as accomplished and talented as Jewel,\" said Paul Telegdy. Jewel previously served as an adviser on Burnett's \"The Voice\" last year. \"Jewel is one of America's greatest singer songwriters and is an amazing storyteller,\" Burnett said. \"...The Sing-Off is sure to be its best [season] yet.\" \"The Sing-Off\" will return in December. See the original story at EW.com.","participant_id":"p01","fixation_seqs":"[1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0, 10.0, 11.0, 12.0, 13.0, 14.0, 15.0, 16.0, 17.0, 18.0, 19.0, 20.0, 21.0, 22.0, 23.0, 24.0, 25.0, 26.0, 27.0, 28.0, 29.0, 30.0, 31.0, 32.0, 33.0, 34.0, 35.0, 36.0, 37.0, 38.0, 39.0, 40.0, 41.0, 42.0, 43.0, 44.0, 45.0, 46.0, 47.0, 48.0, 49.0, 50.0, 51.0, 52.0, 53.0, 54.0, 55.0, 56.0, 57.0, 58.0, 59.0, 60.0, 61.0, 62.0, 63.0, 64.0, 65.0, 66.0, 67.0, 68.0, 69.0, 70.0]","fixation_word_ids":"['62', '20', '9', '1', '3', '5', '6', '7', '12', '19', '20', '20', '21', '5', '5', '4', '40', '77', '77', '99', '123', '122', '123', '100', '12', '19', '20', '36', '50', '19', '26', '20', '20', '21', '32', '51', '50', '26', '26', '28', '41', '58', '46', '47', '62', '61', '75', '77', '88', '109', '106', '121', '123', '123', '123', '128', '.', '.', '.', '.', '.', '.', '13', '7', '5', '6', '123', '.', '82', '51']","fixation_word_texts":"['Bareilles', 'singer-songwriter', 'Context:', 'Claim:', 'sign-Off\"', 'return', 'in', 'last', '\"The', 'Grammy-nominated', 'singer-songwriter', 'singer-songwriter', 'Jewel', 'return', 'return', 'will', 'Stockman', 'Jewel,\"', 'Jewel,\"', 'singer', 'December.', 'in', 'December.', 'songwriters', '\"The', 'Grammy-nominated', 'singer-songwriter', 'Boyz', 'Nick', 'Grammy-nominated', 'cult-favorite', 'singer-songwriter', 'singer-songwriter', 'Jewel', 'winter.', 'Lachey', 'Nick', 'cult-favorite', 'cult-favorite', 'cappella', 'and', 'replaces', 'on', 'the', 'Bareilles', 'Sara', 'talented', 'Jewel,\"', \"Burnett's\", 'Sing-Off', 'Burnett', 'return', 'December.', 'December.', 'December.', 'at', '.', '.', '.', '.', '.', '.', 'Sing-Off\"', 'last', 'return', 'in', 'December.', '.', 'previously', 'Lachey']","fixation_durations":"[135.0, 74.0, 89.0, 164.0, 246.0, 158.0, 177.0, 150.0, 137.0, 33.0, 110.0, 97.0, 173.0, 199.0, 61.0, 188.0, 222.0, 208.0, 186.0, 127.0, 257.0, 51.0, 212.0, 95.0, 128.0, 33.0, 98.0, 325.0, 113.0, 146.0, 52.0, 110.0, 91.0, 205.0, 304.0, 139.0, 299.0, 169.0, 155.0, 46.0, 118.0, 243.0, 125.0, 76.0, 176.0, 192.0, 129.0, 336.0, 173.0, 115.0, 202.0, 83.0, 134.0, 164.0, 77.0, 267.0, 166.0, 80.0, 146.0, 209.0, 168.0, 167.0, 99.0, 159.0, 227.0, 128.0, 25.0, 153.0, 267.0, 182.0]","annotator_labels":"hallucinated","true_labels":"hallucinated"}
```

Comment:
I guess this is a direct copy from one of the dataset files? We shouldn't do that.

Keep the keys of the dictionary but create own dummy data.

Also, if it's a dataset-related test file this should also be visible in the filename.